Creating object script added (#16)
pupilfirst/web-development-101-script: numbers-booleans-objects/creating-and-working-with-objects/README.md

# Script

> In this lesson, we will learn about Javascript Objects. 

Author: Resolved.

> In Javascript, an object is something which is very powerful, and you are going to work with them all the time. So, let's get started.

Reviewer: ```suggestion
In JavaScript, an object is a powerful construct, and you are going to work with them all the time. So, let's get started.
```

Author: Resolved.

### Overview 

> A Javascript object is a collection of named values. Like many other programming languages, it can be compared to objects in real life. Compare it with a cup, for example. A cup is an object, with properties. A cup has a color, a design, weight, a material it is made of. In the same way, JavaScript objects can have properties, which define their characteristics.

Author: Resolved.
